.github/workflows/python-package.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+  schedule:
+  - cron: "0 2 * * 0-6"
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.8]
+        os: [ubuntu-latest, macOS-latest]
```

Comment:
why not Windows?

Reply:
Something weird is with blist on windows. I'll add this back after merging #61.